### PR TITLE
create config-3alpha-to-3

### PR DESCRIPTION
**Pull request by MaazMS**
workshop-operator PROJECT config file has been converted from version 3-alpha to 3.

### CODE REVIEW

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-891f1ffb`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-cdbfb6f6`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-f8a82019`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-34ba7c3d`

File: Makefile [hunk 1]
```diff
@@ -36,7 +36,9 @@ IMAGE_TAG_BASE ?= docker.io/stakater/workshop-operator
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= $(IMAGE_TAG_BASE):v$(VERSION)
+
+IMG ?= quay.io/stakater/workshop-operator:v$(VERSION)
```
Comment (MaazMS) [inline Makefile]: When use `$(IMAGE_TAG_BASE):v$(VERSION)` then it give newTag : controller, that way use base tag image name

File: PROJECT [hunk 2]
```diff
@@ -3,9 +3,15 @@ layout: go.kubebuilder.io/v2
 projectName: workshop-operator
 repo: github.com/stakater/workshop-operator
 resources:
-- group: workshop
+-
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  domain: stakater.com
+  group: workshop
```
Comment (MaazMS) [inline PROJECT:10]: when use command `operator-sdk alpha config-3alpha-to-3` it upadate PROJECT file

File: PROJECT [hunk 3]
```diff
@@ -3,9 +3,15 @@ layout: go.kubebuilder.io/v2
...
   kind: Workshop
+  # TODO(user): Update the package path for your API if the below value is incorrect.
+  path: github.com/stakater/workshop-operator/api/v1
   version: v1
-version: 3-alpha
+version: "3"
```
Comment (MaazMS) [inline PROJECT:15]: To solve issue `Config version 3-alpha has been stabilized as 3, `  There for version update `3-alph` to version `3.

File: bundle/manifests/workshop-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -29,7 +29,8 @@ spec:
       kind: Workshop
       name: workshops.workshop.stakater.com
       version: v1
-  description: A Kubernetes Operator to install the infrastructure for running workshops/bootcamps on OpenShift.
```
Comment (MaazMS) [inline bundle/manifests/workshop-operator.clusterserviceversion.yaml:32]: At time of generate bundle description format is changed.

File: bundle.Dockerfile [hunk 5]
```diff
@@ -1,15 +1,20 @@
 FROM scratch
 
+# Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=workshop-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.3.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.10.0+git
```
Comment (MaazMS) [inline bundle.Dockerfile:9]: update `operators.operatorframework.io.metrics.builder` when project version is updated.

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-93da627f`

File: bundle/manifests/workshop-operator.clusterserviceversion.yaml [hunk 6]
```diff
@@ -259,7 +260,7 @@ spec:
                 - --enable-leader-election
                 command:
                 - /manager
-                image: controller:latest
+                image: quay.io/stakater/workshop-operator:v0.0.1
```
Comment (MaazMS) [inline bundle/manifests/workshop-operator.clusterserviceversion.yaml]: i have pass image name in `makefile` IMG ?= quay.io/stakater/workshop-operator:v$(VERSION)

File: bundle/manifests/workshop.stakater.com_workshops.yaml [hunk 7]
```diff
@@ -20,10 +20,14 @@ spec:
       description: Workshop is the Schema for the workshops API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
           type: string
         kind:
-          description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
```
Comment (MaazMS) [inline bundle/manifests/workshop.stakater.com_workshops.yaml:26]: At time of generate bundle description format is changed.

File: config/manifests/bases/workshop-operator.clusterserviceversion.yaml [hunk 8]
```diff
@@ -15,7 +15,8 @@ spec:
       kind: Workshop
       name: workshops.workshop.stakater.com
       version: v1
-  description: A Kubernetes Operator to install the infrastructure for running workshops/bootcamps on OpenShift.
+  description: A Kubernetes Operator to install the infrastructure for running workshops/bootcamps
```
Comment (MaazMS) [inline config/manifests/bases/workshop-operator.clusterserviceversion.yaml:18]: At time of generate bundle description format is changed.

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-3-54b57825`

File: config/manager/kustomization.yaml [hunk 9]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/stakater/workshop-operator
```
Comment (MaazMS) [inline config/manager/kustomization.yaml]: i have pass image name in makefile IMG ?= quay.io/stakater/workshop-operator:v$(VERSION)

File: bundle/tests/scorecard/config.yaml [hunk 10]
```diff
@@ -12,38 +12,59 @@ stages:
     labels:
       suite: basic
       test: basic-check-spec-test
+    storage:
```
Comment (MaazMS) [inline bundle/tests/scorecard/config.yaml:15]: storage is empty

File: bundle/metadata/annotations.yaml [hunk 11]
```diff
@@ -1,11 +1,14 @@
 annotations:
-  operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  # Core bundle annotations.
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: workshop-operator
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.3.0
```
Comment (MaazMS) [inline bundle/metadata/annotations.yaml:7]: remove operator-sdk-v1.3.0  when update  operator-sdk-v1.11.0

**Review by MaazMS — COMMENTED** [review]
code change review finished

**Pull request merged**